README.md

```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
 an identity provider (IdP) and a service provider (SP).
 
 This charm simplifies configuration of SAML SPs by providing a single point
-of configuration for all the requirers using the same SAML entity. It can be
-deployed on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io)
-to [Charmed Kubernetes](https://ubuntu.com/kubernetes), public cloud Kubernetes
-offerings and virtual machines or bare metal.
+of configuration for all the requirers using the same SAML entity.
 
-As such, the charm makes it easy to manage and propagate SAML configuration, while
-giving the freedom to deploy on the substrate of their choice.
+## Get started
 
-For DevOps or SRE teams this charm will make operating any charm leveraging SAML
-authentication simple and straightforward through Juju's clean interface.
+You can follow the tutorial [here](https://charmhub.io/saml-integrator/docs/tutorial-getting-started).
 
-## Project and community
-
-The SAML Integrator Operator is a member of the Ubuntu family. It's an open source
-project that warmly welcomes community projects, contributions, suggestions,
-fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
-* [Contribute](https://charmhub.io/indico/docs/how-to-contribute)
-Thinking about using the Indico Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
-
-## Contributing to this documentation
-
-Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://charmhub.io/saml-integrator/docs) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, ask a question, or make a suggestion about a potential change via the comments section.
+## Learn more
+* [Read more](https://charmhub.io/saml-integrator)
+* [Developer documentation](https://charmhub.io/saml-integrator/docs)
```

Comment:
I think the developer documentation should be different than our docs. I don't really know of a good external resource for developing SAML (If it even exists).

What do you think? If there's no ideal developer docs for the charm, then I think it's best to remove the line entirely.

Reply:
I'm changing that line for a link to the SAML specification